app/mailers/app_mailer.rb

```diff
@@ -2,21 +2,22 @@ class AppMailer < ActionMailer::Base
 
   default from: 'admin@myflix.com'
 
-  def send_mail_on_register(user)
-    @user = user
-    mail to: user.email, subject: 'You have registered successfully!'
+  def send_mail_on_register(user_id)
+    @user = User.find(user_id)
+    mail to: @user.email, subject: 'You have registered successfully!'
   end
 
-  def send_mail_on_reset_password(user)
-    @user = user
-    @link = reset_password_url(user.reset_password_token, Rails.application.config.action_mailer.default_url_options)
-    mail to: user.email, subject: 'Reset your password now'
+  def send_mail_on_reset_password(user_id)
+    @user = User.find(user_id)
+    @link = reset_password_url(@user.reset_password_token, Rails.application.config.action_mailer.default_url_options)
+    mail to: @user.email, subject: 'Reset your password now'
```

Comment:
I was talking about `to:` not `from:`. You want to send all emails to 1 email address - preferably the one for admin.

```ruby

class AppMailer < ActionMailer::Base
  # ...

  def some_email_sending(user_id)
    @user = User.find user_id
    mail to: filter_recipients(@user.email), # ...
  end

  private

  def filter_recipients(recipients)
    Rails.env.staging? ? "admin@jepetko.at" : recipients
  end
end

```